src/runtime_src/core/pcie/windows/device_windows.cpp

```diff
@@ -97,14 +97,6 @@ rom(const device_type* device, qr_type qr, const std::type_info&, boost::any& va
   case qr_type::QR_ROM_FPGA_NAME:
     value = std::string(reinterpret_cast<const char*>(hdr.FPGAPartName));
     return;
-  }
```

Comment:
Remove this check for device type will not work.  On the user side  there is only limited ROM info, namely the first 4 query values.  Alas, if you ask for QR_ROM_UUID and EPOCH on the user side, then an exception should be thrown.

Reply:
Oh. I see.. I was getting an error with this code on mgmt side. I'll look more closely